pkg/cliutil/install/kubernetes.go

```diff
@@ -60,12 +60,17 @@ func KubectlOut(stdin io.Reader, args ...string) ([]byte, error) {
 		stderr = cliutil.GetLogger()
 	}
 
-	buf := &bytes.Buffer{}
+	stdoutBuf := &bytes.Buffer{}
```

Comment:
The cluster-one e2e tests consistently failed due to data race. E.g: https://github.com/solo-io/gloo/actions/runs/15736563713/job/44395904806#step:7:393 
These changes fixed it.